BusBoard.ConsoleApp/Program.cs

```diff
@@ -1,15 +1,24 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace BusBoard.ConsoleApp
-{
-  class Program
-  {
-    static void Main(string[] args)
+{ 
+    class Program
     {
+        static void Main()
+        {
+            Console.Write("Stop code: ");
+            var stopPoint = Console.ReadLine();
+            var busEntries = ApiHelper.ApiGet<BusEntry>("https://api.tfl.gov.uk/", $"StopPoint/{stopPoint}/Arrivals");
+
+            busEntries.Sort();
+            busEntries = busEntries.GetRange(0, 5);
+
+            foreach (var entry in busEntries)
+            {
+                Console.WriteLine(entry);
+            }
+
+            Console.ReadLine();
```

Comment:
Why do you need this? 🙂

Reply:
It's there to stop the program from closing immediately after it finishes running. Is there a better method I should have used?

Reply:
A continuous loop which asks for new input until the user enters quit seems reasonable? 🙂